### ISSUE TITLE
Insights -- What could be wrong

### ISSUE THREAD

**Issue by ma7555** [ozora-ogino/wav2spk#3]
So, first of all thanks for updating the code i will raise some insights that could be meaningful:

1. Why are you using axis=1 for InstanceNormalization instead of -1? 
2. In the feature encoder, using 'same' padding for such a large kernel and stride will result in A LOT of padding. I don't think it's beneficial at all. So assuming 20,000 as sample length, around 1.25 seconds of 16K sampling rate wav, after the first convolutional layer the output is as follows (kernel_size=10, strides=5)
```Layer (type)                    Output Shape         Param #     Connected to
==================================================================================================
raw_audio (InputLayer)          [(None, 20000, 1)]   0
__________________________________________________________________________________________________
conv_1 (Conv1D)                 (None, 3999, 40)     440         raw_audio[0][0]
```
so padding 3999 to 20000 is adding way too much noise to the model.

These are my insights

**Comment by ozora-ogino** [comment]
@ma7555 , Your insights are correct.
I fixed both. Thank you for your help.

**Comment by ma7555** [comment]
Model is training now for me, but results are far from the paper. maybe we should start seeing the am-softmax and stats pooling.
